xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -292,7 +292,8 @@ ASS_Image* CDVDSubtitlesLibass::RenderImage(double pts,
   {
     ass_set_storage_size(m_renderer, static_cast<int>(opts.sourceWidth),
                          static_cast<int>(opts.sourceHeight));
-    useFrameMargins = opts.marginsMode != MarginsMode::DISABLED;
+    useFrameMargins =
+        opts.marginsMode == MarginsMode::DISABLED || opts.marginsMode == MarginsMode::INSIDE_VIDEO;
```

Comment:
Is the change intentional? Before you had != Marginsmode::disabled and now you have ==

Reply:
yes since variables on OverlayRender are changed we have
DISABLED only when overrides are disabled
INSIDE_VIDEO when overrides are enabled with position inside video

so in this case we need to enable it when set as disabled
otherwise as alternative, maybe better, we need to rename `MarginsMode::DISABLED` to something like `MarginsMode::NATIVE`